pkg/api/handlers/compat/images.go

```diff
@@ -259,10 +259,12 @@ func CreateImageFromImage(w http.ResponseWriter, r *http.Request) {
 
 	query := struct {
 		FromImage string `schema:"fromImage"`
-		Tag       string `schema:"tag"`
 		Platform  string `schema:"platform"`
+		TLSVerify bool   `schema:"tlsVerify"`
```

Comment:
I don't see the Docker API having that parameter.

Reply:
@rhatdan the Docker API doesn't have such a parameter.  I don't think this PR is needed.